labs/shipped-import-project/1.md

```diff
@@ -0,0 +1,34 @@
+# Shipped - Import Existing project to shipped
+
+1 Choose Exiting Project
+You can select your git repo you want to integrate with shipped, For this example we have sample project 'goproject' written in golang. 
+![](assets/1.PNG)
+
+
+To Integrate your project with shipped you need to have Docker file. this docker file content varies depend on your project requirement.
+
+2 Create a Docker file- This section is specific to the type of project (this project is in Golang)
+
+For Example:
```

Comment:
Add inline comment in docker file with # 
I think // is not supported in Dockerfile commenting. 
Remove image for dockerfile.